distribution/openrct2.d.ts

```diff
@@ -2606,6 +2606,51 @@ declare global {
         "waiting_to_depart" |
         "waiting_to_start";
 
+    type CrashParticleType =  0 | 1 | 2 | 3 | 4;
```

Comment:
Would it be nicer to make this a string union/enum? Or are the different types not really name-able?

Reply:
The differences are in the shape and spin angle, which isn't easy to describe but doable e.g. "crashparticle0"

Reply:
If it provides no further clarity then the numbers are fine IMHO. 🙂

Reply:
@spacek531 After testing them a bit, they do seem to be recognisable. We could name them like:
```
type CrashParticleType =  "corner" | "rod" | "wheel" | "panel" | "seat";
```

Reply:
Did you mark this as resolved? It says I did but I don't remember doing so 😵‍💫

Reply:
I didn't and I can't. 😛 I can imagine you resolved it because initially I said the numbers were fine.